### PR TITLE
[22407] Cypress Testing Facilties Page

### PR DESCRIPTION
**Pull request by LanceGundersen**
## Description
Run axe checks on both variations of the contact facilities page in the vaccine flow.

## Testing done
E2E

## Screenshots
image

## Acceptance criteria
- [ ] Cypress test covers when a user chooses "Yes" on the have you received a dose page and the axe check is run on the contact facilities page
- [ ] Cypress test covers when vaccine schedule is not available for that user and they are sent to the contact facilities page

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/tests/e2e/vaos-cypress-helpers.js [hunk 1]
```diff
@@ -644,8 +644,20 @@ export function initVAAppointmentMock({ cernerUser = false } = {}) {
   mockSubmitVAAppointment();
 }
 
-export function initVaccineAppointmentMock() {
+export function initVaccineAppointmentMock(unableToScheduleCovid = false) {
```
Comment (jbalboni) [inline src/applications/vaos/tests/e2e/vaos-cypress-helpers.js:647]: I think for these helpers making the params an object makes the tests clearer. So like: `initVaccineAppointmentMock({ unableToScheduleCovid = false } = {})` and then when called it's `initVaccineAppointmentMock({unableToScheduleCovid: true })`.

**Review by jbalboni — APPROVED** [review]
One really minor comment, looks good otherwise

File: src/applications/vaos/tests/e2e/vaos-cypress-helpers.js (hunk 1 above)
Comment (LanceGundersen) [reply src/applications/vaos/tests/e2e/vaos-cypress-helpers.js:647]: Interesting, yes that is much clearer to read.

Comment (LanceGundersen) [reply src/applications/vaos/tests/e2e/vaos-cypress-helpers.js]: Suggestion added

**Review by vbahinwillit — APPROVED** [review]

**Pull request merged**